### ISSUE TITLE
Support markdown syntax

### ISSUE THREAD

**Issue by iaguirre88** [iaguirre88/reading_time#1]
Ignore markdown syntax when counting words so the estimation is more accurate

**Comment by joseemds** [comment]
Hi! can i work on this?

**Comment by iaguirre88** [comment]
Hi @uduDudu 

I've been thinking a while about this feature and I'm not sure it worths the effort

I'm not sure I can think of a real use case for this and also this library only has 22 downloads so far.

However, if you are interested in the technical challenge you are more than welcome to give it a try

**Comment by joseemds** [comment]
Yeah, im actually learning elixir and i think this is a good challengr, will give it a try
